### ISSUE TITLE
YAML frontmatter triggers warning for "Multiple top-level headings in the same document"

### ISSUE THREAD

**Issue by Alterae** [DavidAnson/vscode-markdownlint#134]
I have a markdown file exported from Scrivener which begins as such:

```markdown
---
title: aFileName
author: (My name)
---

# Top-Level Heading

A bunch of text follows...
```
The heading on line six triggers this warning, `MD025/single-title/single-h1: Multiple top level headings in the same document`, despite the fact that there are no other top-level headings in the document.  Deleting the YAML frontmatter causes this warning to go away.

**Comment by DavidAnson** [comment]
This is explained in the documentation for MD025:

> If YAML front matter is present and contains a title property (commonly used with blog posts), this rule treats that as a top level heading and will report a violation for any subsequent top level headings. To use a different property name in front matter, specify the text of a regular expression via the front_matter_title parameter. To disable the use of front matter by this rule, specify "" for front_matter_title.

Or you can disable the rule entirely.

https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md025
